app/models/role.rb

```diff
@@ -171,6 +175,7 @@ def clone(role_params = {})
   end
 
   def locked?
+    return false if self.class.bypass_locking
```

Comment:
If we extend the existing `not_locked` check, it still fails. The reason is that `Filter#role_not_locked` would also need to be aware of `Role.bypass_locking`